main_logic/core.py

```
@@ -2032,28 +2168,36 @@ async def feed_tts_chunk(self, text: str):
+                        self.session._conversation_history.append(_AIMsg(content=full_text))
 
-            if self.use_tts and self.tts_thread and self.tts_thread.is_alive():
-                try:
-                    self.tts_request_queue.put((None, None))
-                except Exception:
-                    pass
+                    if self.use_tts and self.tts_thread and self.tts_thread.is_alive():
+                        try:
+                            self.tts_request_queue.put((None, None))
+                        except Exception:
+                            pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)